### DISCUSSION TITLE
SDK 1.9 BT-06-Lot and BT-777-Lot Mandatory ?

### DISCUSSION THREAD

**Discussion by rlagrange98** [OP-TED/eForms-SDK#1055] [Q&A] [answered]
Hello,

Currently with 1.9 if you don't fill the fiedls BT-06-Lot and BT-777-Lot you have an error message and you can't send your form :
![image](https://github.com/user-attachments/assets/1bc2f79c-714c-489d-a570-97ecb170adbf)

Is it a normal behavior or not ?
Also do you have to fill the two fields or just one of them to bypass this issue ?

Best regards,
Romain Lagrange

**Comment by YvesJo** [comment, marked as answer]
Hi,

The full path for BT-06 is:

`/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:ProcurementProject/cac:ProcurementAdditionalType[cbc:ProcurementTypeCode/@listName='strategic-procurement']/cbc:ProcurementTypeCode`

and the one for BT-777:

`/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:ProcurementProject/cac:ProcurementAdditionalType[cbc:ProcurementTypeCode/@listName='strategic-procurement']/cbc:ProcurementType`

and these do not match the one in the message which is about a component of the Award Criteria

KR
